uint16 target_point_x
uint16 target_point_y
uint16 lower_slice_row
uint16 upper_slice_row
uint16 slice_height
uint16[] lower_lane_boundaries
uint16[] upper_lane_boundaries
bool crossing_detected
uint16 ts_x
uint16 ts_y
uint16 ts_w
uint16 ts_h
uint16 ts_type
uint16 stop_registered
uint16 direction
uint16 speed



